client/src/components/Form/InputDisplay/InputDisplay.tsx

```diff
@@ -8,20 +8,33 @@ import { ReactComponent as Edit } from '../formIcons/edit.svg';
 import { ReactComponent as Visible } from '../formIcons/visible.svg';
 import { ReactComponent as Invisible } from '../formIcons/invisible.svg';
 
-import { Col } from 'react-bootstrap';
+import Col from 'react-bootstrap/Col';
+import { DisableInput } from '../../../enums/DisableInput';
 
-const InputDisplay = (props) => {
-    const [show, setShow] = useState(false);
+interface InputDisplayProps {
+    inputTitle: string,
+    inputValue: string,
+    inputId: string,
+    inputValueSave: (id: string, value: string) => void,
+    inputType?: string,
+    inputFieldAsSelect?: boolean,
+    optionsForSelect?: Array<string>,
+    textForSelect?: Array<string>,
+    inputVisibility?: (inputId: string) => void,
+    inputVisible: boolean,
+    toDisable?: DisableInput,
+    validation?: ValidationRuleType,
+    format?: string
+};
```

Comment:
We should reconsider the names of these variables to express what they really do or cointain. like setInputVisibility

Reply:
You are right Kristof. I can imagine here a better variable name, like: set-, change-, handle-InputVisibility.
On the other hand IntelliSense explicitly presents this props as a function. So deducting the usage is easy IMHO.
As a general rule I do not change variable names on this branch, in case we need to compare before and after operation for debugging, the task is easier this way.